Makefile.doc

```diff
@@ -251,29 +251,31 @@ $(DOC_STDLIB_DIR)/FullLibrary.pdf: $(DOCCOMMON) $(DOC_STDLIB_DIR)/FullLibrary.co
 .PHONY: install-doc install-doc-meta install-doc-html install-doc-printable \
 	install-doc-sphinx install-doc-stdlib-html
 
+DOCDIRDEST=$(DESTDIR)/$(DOCDIR)
```

Comment:
You made an argument in the other file that this is wrong for empty DESTDIR on Windows, no?

Reply:
Oh yes sorry that slipped my fix for the windows path, thanks for the catch!

Reply:
fixed